.github/workflows/screener-build.yml

```diff
@@ -0,0 +1,170 @@
+name: Screener build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  TestJob:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Hello there"
+
+  EnvironmentUpload:
+    runs-on: ubuntu-latest
+    steps:
+      - run: mkdir artifacts
+
+      ###########################################
+      # BROWSERSLIST_IGNORE_OLD_DATA = Prevents failures on CI when "caniuse-lite" becomes outdated
+      # DEPLOYHOST = address of host for screener tests deployment
+      # BUILD_BUILDID = unique ID of the workflow run within the repo
+      # BUILD_SOURCEBRANCH = branch that triggered the workflow -> refs/pull/<pr_number>/merge
+      ###########################################
+
+      - name: Set base env variables
```

Comment:
Let's also some documentation why we write env variables as an artifact. Also mention that this is temporary and link your github issue in the comment